packages/notifi-react-card/README.md

```diff
@@ -590,3 +590,103 @@ export const Notifi: React.FC = () => {
 ```
 
 </details>
+
+### WalletConnect (Cross-chain wallet adaptor)
+
+<details>
+<summary>Integrate Card Component</summary>
+
+Note:
+
+- Ethers.js & wagmi are used. Be sure these two are installed as dependencies.
+- `NotifiContext` params needs to be updated accordingly.
+
+Create a WallectConnectProvider by WagmiConfig
+
+```tsx
+import { FC, PropsWithChildren } from 'react';
+import { WagmiConfig, configureChains, createClient, mainnet } from 'wagmi';
+import { WalletConnectConnector } from 'wagmi/connectors/walletConnect';
+import { infuraProvider } from 'wagmi/providers/infura';
+
+export const connector = new WalletConnectConnector({
+  chains: [mainnet],
+  options: {
+    projectId: '<YOUR WALLETCONNECTOR PROJECT ID HERE>', // Get Project ID at https://cloud.walletconnect.com/
```

Comment:
```suggestion
    projectId: '<YOUR WALLETCONNECTOR PROJECT ID HERE>', // Get Project ID at https://cloud.walletconnect.com/
```

Should it be?

    projectId: '<YOUR WALLETCONNECT PROJECT ID HERE>', // Get Project ID at https://cloud.walletconnect.com/